recipes/mds.rb

```diff
@@ -23,19 +23,22 @@
 # cluster = 'ceph'
 cluster = node['ceph']['cluster']
 
-if node['ceph']['version'] == 'hammer'
-  directory "/var/lib/ceph/mds/#{cluster}-#{node['hostname']}" do
-    owner node['ceph']['owner']
-    group node['ceph']['group']
-    mode node['ceph']['mode']
-    recursive true
-    action :create
-    not_if "test -d /var/lib/ceph/mds/#{cluster}-#{node['hostname']}"
-  end
+directory "/var/lib/ceph/mds/#{cluster}-#{node['hostname']}" do
+  owner node['ceph']['owner']
+  group node['ceph']['group']
+  mode node['ceph']['mode']
+  recursive true
+  action :create
+  not_if "test -d /var/lib/ceph/mds/#{cluster}-#{node['hostname']}"
```

Comment:
I think the `:create` action already tests something similar to this, is there a reason to have this explicitly specified?

Reply:
I just used the same code as before and removed it from the block.